README.md

```diff
@@ -214,10 +215,59 @@ index 68e73a1c8..300aa3d80 100644
 ```
 
 #### Run
-Start the docker container:
+Start the Docker container:
 
 ```sh
 docker-compose up
 ```
 
 Navigate to the exposed `localhost` endpoint.
+
+## Docker Build For Participants
+The [`materials/Dockerfile`](https://github.com/stratum-mining/sv2-workshop/blob/main/materials/Dockerfile)
+contain the Docker image with the following installed, configured, and built:
+
+1. [Plebhash's branch of Sjors's `sv2-tp-0.1.3`](https://github.com/plebhash/bitcoin/releases/tag/btc-prague): Used for the Pool and Miner Roles.
```

Comment:
my fork can be completely removed from the workshop

see https://github.com/stratum-mining/sv2-workshop/pull/8/files#r1679286240